gtfs-realtime/spec/en/vehicle-positions.md

```diff
@@ -1,6 +1,6 @@
 Vehicle position is used to provide automatically generated information on the location of a vehicle, such as from a GPS device on board. A single vehicle position should be provided for every vehicle that is capable of providing it.
 
-The trip that the vehicle is currently serving should be given through a [trip descriptor](reference.md#TripDescriptor). You can also provide a [vehicle descriptor](reference.md#VehicleDescriptor), which specifies a precise physical vehicle that you are providing updates about. Documentation is provided below.
+The trip that the vehicle is currently serving should be given through a [trip descriptor](reference.md#message-tripdescriptor). You can also provide a [vehicle descriptor](reference.md#message-vehicledescriptor), which specifies a precise physical vehicle that you are providing updates about. Documentation is provided below.
```

Comment:
> To match the formatting in the trip-updates.md docs, we should probably change "trip descriptor" to "TripDescriptor" and "vehicle descriptor" to "VehicleDescriptor" in the text.

+1

> I've noticed that we also don't standardize on using the code formatting ticks or using normal formatting, although looks like it's mostly StopTimeUpdate (and not StopTimeUpdate) in trip-updates.md. Probably outside the scope of this PR but it would be nice to be consistent throughout.

Also +1, but I agree that would be better kept for another PR. While we're on the topic I'll add something else to the laundry list of formatting improvements that I noticed during this PR: 

The same anchor link appears multiple times in the reference. So that when we want to anchor link to TripDescriptor and VehicleDescriptor under the VehiclePositions element, it defaults to those with the same names as they first appear in the reference under the FeedEntity element. This is a limitation of Markdown, so some sort of semantic hierarchy would be needed to make these anchor links work properly.

Reply:
I applied the styling on all elements where it was missing. Let me know if it LGTY.